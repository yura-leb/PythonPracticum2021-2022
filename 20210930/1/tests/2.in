89, 100, 23, 56
