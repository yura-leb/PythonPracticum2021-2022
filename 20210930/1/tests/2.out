[100, 89, 23, 56]
